packages/makeswift/src/integrate-next-app.ts

```diff
@@ -5,8 +5,40 @@ import path from 'path'
 import { manipulateNextConfig } from './utils/manipulate-next-config'
 import { yarnOrNpm } from './utils/yarn-or-npm'
 import { createFolderIfNotExists } from './utils/create-folder-if-not-exists'
+import inquirer from 'inquirer'
+import chalk from 'chalk'
+
+async function getApprovalToIntegrate(dir: string): Promise<boolean> {
+  const projectName = dir.split('/').reduce((prev, curr) => curr)
+  return new Promise(resolve => {
+    const questions = [
+      {
+        type: 'confirm',
+        name: 'approval',
+        default: false,
+        message: `It appears ${chalk.cyan(
+          projectName,
+        )} is an existing Next.js app — would you like to integrate it?`,
+      },
+    ]
+
+    inquirer.prompt(questions).then(answers => {
+      if (typeof answers.approval == 'boolean') {
+        resolve(answers.approval)
+      } else {
+        throw Error('Something went wrong')
```

Comment:
Do we want a better error message here? I'm assuming this will be caught and the message will be presented to the user?

Reply:
Yeah, going to do more more work on error messages and how that's presented to the user in another PR